src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -510,7 +510,7 @@ public GHPullRequestReviewComment createReviewComment(String body, String sha, S
                 .with("body", body)
                 .with("commit_id", sha)
                 .with("path", path)
-                .with("position", position)
+                .with("line", position)
```

Comment:
This too needs to be a separate PR, with tests for the change.

The docs indicate that `position` and `line` are not equivalent. You'll probably need to implement a builder to support the various parameter combinations.
https://docs.github.com/en/free-pro-team@latest/rest/reference/pulls#create-a-review-comment-for-a-pull-request